src/test/resources/pages/frame-page.html

```diff
@@ -0,0 +1,17 @@
+<html xmlns:v="urn:schemas-microsoft-com:vml"
+xmlns:o="urn:schemas-microsoft-com:office:office"
+xmlns:x="urn:schemas-microsoft-com:office:excel"
+xmlns="http://www.w3.org/TR/REC-html40">
+
+<head>
+</head>
+
+<frameset rows="*,39" border=0 width=0 frameborder=no framespacing=0>
+ <frame src="frame-diff.html" name="frSheet">

```

Comment:
Do we really need a unnecessary large page as frame with 2.932 lines of code, which in turn produces a 55.000 line Golden Master? I would suggest to simplify this, since it is not really the exported excel from our customer that is the problem, but moreover the cascading frames.